### DISCUSSION TITLE
Notification Bar Title

### DISCUSSION THREAD

**Discussion by grahampugh** [IBM/mac-ibm-notifications#10] [Q&A] [answered]
Hi, this app looks great! Is the notification "bar title" customisable? The image in the wiki suggests not. It would be difficult for us to use notifications that said "MAC@IBM NOTIFICATIONS" on it. I guess we could recompile it?

**Comment by SMartorelli** [comment]
Hello Graham, yes of course you can customise pop-up notification's bar title using the `-bar_title` argument. You can read more about on the [pop-up UI related section](/https://github.com/IBM/mac-ibm-notifications/wiki/Usage#pop-up-ui-additional-arguments) of our "Usage" wiki page. If you're referring to the Notification Center banner title it is not customisable because it always shows the "app name", in this case "Mac@IBM Notifications". You should rebrand the app in order to change it.

**Comment by grahampugh** [comment, marked as answer]
Hi @SMartorelli
I'm talking about the Notifications rather than the pop-ups. In the attached screenshot you have highlighted area that you say are customisable. But what about the top part that says `MAC@IBM NOTIFICATIONS`? I'm guessing this is related to the name of the app, and if the name cannot be customised, then I guess this cannot either?

![](https://github.com/IBM/mac-ibm-notifications/raw/main/Images/Banner.png)

**Reply by SMartorelli** [reply]
Yes it refers to the app name, feel free to fork the project and rebrand the app for your usage!

**Reply by grahampugh** [reply]
Thanks for the confirmation, that's what I expected but just wanted to confirm. ☑️
